docs/TABBEDHEADER.MD

```diff
@@ -0,0 +1,382 @@
+# Tabbed Header Customisation
+
+Example of custom Tabbar Header styling. Whole source code can be found in the summary of this page.
+
+![Tabbed Header Gif](../assets/readme_yoda.gif)
+
+## Custom scrollable tabs
+
+To change tabs to your custom ones, pass `tabs` prop to StickyParallaxHeader component, where
+`title` is your tab name and `content` is a component to render for specific tab:
+
+```jsx
+tabs={[
+  {
+    title: 'Biography',
+    content: renderContent(text)
+  },
+  {
+    title: 'Powers and Abilities',
+    content: renderContent(text1)
+  },
+  {
+    title: 'Appearances',
+    content: renderContent(text2)
+  }
+]}
+```
+
+Example of such component:
+
+```jsx
+const windowHeight = Dimensions.get('window').height
+
+const styles = StyleSheet.create({
+  contentContiner: {
+    height: windowHeight,
+    padding: 10
+  },
+  contentText: {
+    fontSize: 16
+  }
+})
+
+const renderContent = x => (
+  <View
+    style={styles.contentContiner}>
+    <Text style={styles.contentText}>{x}</Text>
+  </View>
+)
+
+```
+
+## Custom tab styling
+
+When It comes to tab styling, in TabbedHeader there are seven props:
+
+`tabText` and `tabTextActiveStyle` to pass styles for tab text wheter is inactive or active
```

Comment:
is it tabText or tabTextStyle? It seems like all other visual props have style/color etc. Does this one not have it?

Reply:
changed prop name